public-docs/swag-shop-05-custom-home-page.md

```diff
@@ -0,0 +1,106 @@
+---
+id: swag-shop-5
+title: Adding a custom homepage
+---
+
+# Adding a custom home page
+
+By default the home page for the starter-kit just shows a grid of products, but some people will probably want some custom content. So let's create a new component in the `custom` directory with just some placeholder text for now. So it would look something like this:
+
+```javascript
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+
+
+const styles = (theme) => ({
+  homepage: {
+    alignItems: "center",
+    display: "flex",
+    justifyContent: "center",
+    marginBottom: theme.spacing.unit * 2
+  }
+});
+
+const HomePage = () => (
+  <Typography variant="caption">
+    <div className="homepage">
```

Comment:
The class name should come from `props.classes.homepage` as the className will be made unique with some extra modifiers.

Also. We should always promote the use of the component classes vs the stateless function components. There's no performance difference, and it helps the docs be more consistent.